.github/workflows/reusable_python_build.yml

```diff
@@ -38,6 +38,15 @@ jobs:
       if: ${{ inputs.commit }}
       with:
         ref: ${{ inputs.commit }}
+      
+    - name: setup ${{ inputs.os }}
+      if: inputs.os == 'ubuntu-latest'
+      run: |
+        sudo apt-get install -y libegl1 libdbus-1-3 libxkbcommon-x11-0 \
```

Comment:
we should run `sudo apt-get update` first.

Reply:
yep, updated